crates/proof-of-sql/src/base/database/column.rs

```diff
@@ -144,6 +144,13 @@ impl<'a, S: Scalar> Column<'a, S> {
         }
     }
 
+    /// Generate a `Int128` `rho` column [0, 1, 2, ..., length - 1]
+    pub fn rho(length: usize, alloc: &'a Bump) -> Self {
+        let raw_rho = (0..length as i128).collect::<Vec<_>>();
```

Comment:
Using `i128` since otherwise we will get the clippy warning for `usize` -> `i64` can lead to wrapping